### ISSUE TITLE
Snowballs sometimes are stuck on the corner

### ISSUE THREAD

**Issue by HybridDog** [Splizard/minetest-mod-snow#36]
![screenshot_20180320_141515](https://user-images.githubusercontent.com/3192173/37656742-35847080-2c49-11e8-8303-553f11b60bab.png)

**Comment by Splizard** [comment]
Yes I have seen this happen, maybe have a life timer? When the snowball exists for too long.. remove it.

**Comment by HybridDog** [comment]
When the snowball has the exact same position in two consecutive on_step executions, a small random velocity could be added to make it moving again.

**Comment by HybridDog** [comment]
I've pushed a fix: https://github.com/Splizard/minetest-mod-snow/commit/11364a550c7439a091b9eb8666eb269f58b419cf
